addons/Assessments_Navigation_Bar/src/style.css

```diff
@@ -22,15 +22,24 @@
 }
 
 .assessments-navigation-bar-wrapper .sections {
-    display: inline-block;
+    display: flex;
```

Comment:
Obawiam się, że może to popsuć istniejący content. Tj. modyfikujesz style, dla elementów które już istnieją. Najlepiej byłoby dodać dodatkową klasę na to.